src/azure-cli/azure/cli/command_modules/netappfiles/_help.py

```diff
@@ -338,6 +338,14 @@
     short-summary: The ARM Id or name of the subnet for the vnet. If omitted 'default' will be used
   - name: --protocol-types
     short-summary: Space seperated list of protocols that the volume can use
+  - name: --volume-type
+    short-summary: Whether the volume should be a data protection volume ("DataProtection")
```

Comment:
Which are available volume types? Can you show up them in help message?

Reply:
At this time this can be left empy or set to "DataProtection".  There are plans to change this to an enum in a later version.